packages/test/src/workflows/log-before-timing-out.ts

```diff
@@ -1,10 +1,9 @@
-import { proxySinks } from '@temporalio/workflow';
-import { LoggerSinks } from './definitions';
+import { LoggerSinks, proxySinks } from '@temporalio/workflow';
 
-const { logger } = proxySinks<LoggerSinks>();
+const { defaultWorkerLogger } = proxySinks<LoggerSinks>();
 
 export async function logAndTimeout(): Promise<void> {
-  logger.info('logging before getting stuck');
+  defaultWorkerLogger.info('logging before getting stuck');
```

Comment:
Why not use `workflow.log`? Don't want the extra context?

Reply:
I did exactly that at some point, but ended reverting that back.

The thing that was bugging me was that log entries captured by the default logger included default lifecycle log messages, emmited from the interceptor. Added some logic to filter them out, but that only made the tests even more complex. Concluded that using a custom sink for that just the simplest approach for that specific case. Plus, having tests for _custom_ sinks is desirable.